demo/examples/src/wallet_connect.cc

```diff
@@ -139,10 +111,14 @@ int main(int argc, char *argv[]) {
         if (test_personal) {
             /* message signing */
             rust::Vec<uint8_t> sig1 = client->sign_personal_blocking(
-                "hello", result.addresses[0].address);
+                "hello", result.eip155.accounts[0].address.address);
```

Comment:
is the first address here an object that contains the value for address? address.address seems strange

Reply:
yes, i'll add comment